modules/dreamview/proto/camera_update.proto

```diff
@@ -13,4 +13,8 @@ message CameraUpdate {
 
     // Camera image data
     optional bytes image = 3;
+
+    // Camera image width and height
+    optional uint32 img_height = 4;
+    optional uint32 img_width = 5;
```

Comment:
Why not simply set the aspect ratio here as img_height and img_width are used for calculating aspect ratio only.